### PR TITLE
Readme Update setup instructions

### PR DESCRIPTION
**Pull request by oshhh**
Updated readme setup instructions: made them more detailed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,9 +7,9 @@ to enable home insurance providers give discounts to home owners if they have ce
...
-2. Make sure flutter plugin and dart plugin has been installed on android studio. If not go to Help->Find Action->plugins and install flutter plugin.
-3. Clone the repository and open the home_insurance_app from android studio
+1. Run flutter doctor to check if flutter is installed in your laptop. If not, please refer to https://flutter.dev/docs/get-started/install for installation instructions. 
+2. Go to Find Action -> plugins -> installed. if installed shows dart and flutter then continue to next step, else go to Find Action -> plugins -> marketplace and search for flutter and add the plugin. If prompted to install dart plugin as well, accept. Otherwise search for dart plugin and install that as well. 
+3. Clone the repository and open the 'home_insurance_app'/'sdk' directory from android studio not the root directory
 4. Go to settings -> flutter and add the path to flutter SDK if not already present.
-5. Connect device/ emulator and run the app.
-
+5. If prompted to run 'flutter pub get' accept, else go to terminal in the IDE and run flutter pub get.
+5. Connect device/emulator and run the app.
```
Comment (dreamplay) [inline README.md:15]: can you add more details on step 5.

File: README.md [hunk 2]
```diff
@@ -7,9 +7,9 @@ to enable home insurance providers give discounts to home owners if they have ce
...
-1. Make sure flutter SDK is installed on your laptop.
-2. Make sure flutter plugin and dart plugin has been installed on android studio. If not go to Help->Find Action->plugins and install flutter plugin.
-3. Clone the repository and open the home_insurance_app from android studio
+1. Run flutter doctor to check if flutter is installed in your laptop. If not, please refer to https://flutter.dev/docs/get-started/install for installation instructions. 
+2. Go to Find Action -> plugins -> installed. if installed shows dart and flutter then continue to next step, else go to Find Action -> plugins -> marketplace and search for flutter and add the plugin. If prompted to install dart plugin as well, accept. Otherwise search for dart plugin and install that as well. 
+3. Clone the repository and open the 'home_insurance_app'/'sdk' directory from android studio not the root directory
 4. Go to settings -> flutter and add the path to flutter SDK if not already present.
-5. Connect device/ emulator and run the app.
-
+5. If prompted to run 'flutter pub get' accept, else go to terminal in the IDE and run flutter pub get.
```
Comment (dreamplay) [inline README.md:14]: how can one install pub, maybe add a command for that or a link

File: README.md [hunk 3]
```diff
@@ -7,9 +7,9 @@ to enable home insurance providers give discounts to home owners if they have ce
 **This is not an officially supported Google product.**
 
 **To setup flutter app on android studio**:
-1. Make sure flutter SDK is installed on your laptop.
-2. Make sure flutter plugin and dart plugin has been installed on android studio. If not go to Help->Find Action->plugins and install flutter plugin.
-3. Clone the repository and open the home_insurance_app from android studio
+1. Run flutter doctor to check if flutter is installed in your laptop. If not, please refer to https://flutter.dev/docs/get-started/install for installation instructions.
```
Comment (dreamplay) [inline README.md:10]: can we have 2 sections, if required, for linux and mac.
if same works for both, maybe mention it at top.

Comment (oshhh) [reply README.md:10]: the official link shared has separate instructions for linux and mac.

File: README.md (hunk 2 above)
Comment (oshhh) [reply README.md:14]: I didn't have to install pub separately. Should be installed with flutter I guess.

File: README.md (hunk 1 above)
Comment (oshhh) [reply README.md]: added

File: README.md (hunk 2 above)
Comment (oshhh) [reply README.md:14]: I don't think its supposed to be installed separately. You just have to install flutter.

File: README.md [hunk 4]
```diff
@@ -7,9 +7,10 @@ to enable home insurance providers give discounts to home owners if they have ce
 **This is not an officially supported Google product.**
 
 **To setup flutter app on android studio**:
-1. Make sure flutter SDK is installed on your laptop.
-2. Make sure flutter plugin and dart plugin has been installed on android studio. If not go to Help->Find Action->plugins and install flutter plugin.
-3. Clone the repository and open the home_insurance_app from android studio
+1. Run flutter doctor to check if flutter is installed in your laptop. If not, please refer to https://flutter.dev/docs/get-started/install for installation instructions. 
+2. Go to Find Action -> plugins -> installed. if installed shows dart and flutter then continue to next step, else go to Find Action -> plugins -> marketplace and search for flutter and add the plugin. If prompted to install dart plugin as well, accept. Otherwise search for dart plugin and install that as well. 
+3. Clone the repository and open the 'home_insurance_app'/'sdk' directory from android studio not the root directory
```
Comment (dreamplay) [inline README.md:12]: why is step 3 to be done for sdk directory and not directly on project level directory?

**Comment by dreamplay** [comment]
please resolve cla issue

File: README.md (hunk 4 above)
Comment (oshhh) [reply README.md:12]: when running the home insurance app it looks for pubspec.yaml in root and if it is open from one directory above it is not easy to set it up for running. Didn't work for me.

**Comment by oshhh** [comment]
@googlebot I consent.

**Comment by anjalisoni3655** [comment]
@googlebot I consent.

**Pull request merged**